lib/models/question_model.dart

```diff
@@ -1,23 +1,37 @@
 class QuestionModel {
-  final int id;
+  final String id;
   final String title;
-  final DateTime date;
+  final String questionDate;
   final String description;
-  final List<String> mediaUrls;
   final String category;
-  final int status;
-  final int visible;
-  final String uuid;
+  // for Lists (both for list of string and list of integers), use List<dynamic>, e
+  final List<dynamic> mediaUrls;
+  final bool status;
+  final bool visible;
+  final String userId;
 
-  QuestionModel({
-    this.id,
-    this.title,
-    this.date,
-    this.description,
-    this.mediaUrls,
-    this.category,
-    this.status,
-    this.visible,
-    this.uuid,
-  });
+  QuestionModel(
+      {this.id,
+        this.title,
+        this.questionDate,
+        this.description,
+        this.category,
+        this.mediaUrls,
+        this.status,
+        this.visible,
+        this.userId});
+
+  factory QuestionModel.fromJson(Map<String, dynamic> json) {
+    return QuestionModel(
+      id: json['id'],
+      title: json['title'],
+      questionDate: json['questionDate'],
+      description: json['description'],
+      category: json['category'],
+      mediaUrls: json['mediaUrls'],
+      status: json['status'],
+      visible: json['visible'],
+      userId: json['userId'],
+    );
+  }
```

Comment:
For questions model I would auto set the time/date for questionDate parameter and remove it from postman body request. That way the user and testers don't mess up the date and time of the questions they post.

Reply:
This is unrelated to Max's task. Max is only getting data, not sending any data.

Reply:
This model is used not only to to send data to Backend, but to receive the data. `questionDate` is a timestamp that was pulled from the database and sent to Flutter. 
In case with sending POST request, it is possible that Spring Boot could create a timestamp instead of Flutter and store it in the database without receiving timestamp attribute from Flutter, but currently we don't have this functionality, and I am not sure if that is a good practice to run this operation in Backend, but I might be wrong.

Reply:
It can work both ways in this case, and doing it in the front end in this case might be better since it'll be a bit more accurate to when the user actually creates it, rather then the user uploads it and then wait slight lag when it's sent to the backend to generate the time. It's microseconds difference and won't matter too much, but I think front end is better.

Reply:
Good point, I agree.